research/CongressKeynote/Congress-Keynote.md

```diff
@@ -0,0 +1,466 @@
+
+---
+
+## Slide 36: Detecting cryptic species
+{% include figure.html src="Slides/Slide39.png" caption="Slide 39." %}
+
+Perhaps the biggest surprise we have had in the QUT Ecoacoustics Group recently is the usefulness of generic acoustic indies in detecting cryptic and/or threatened bird species such as the Lewin's Rail, Black Rail, Least Bittern and Bhutanese White Heron.
+
+The advantages of this technique are:
+1: Compressed timescale; 2: Generic features; 3: get soundscape + species detection
+
+- Faster preparation of datasets because of labelling at low resolution low resolution, i.e. one label per one minute.
+
+- Faster preparation of recognisers because of using generic indices.
+
+- Get both soundscape visualisation and species recogniser in the same processing steps.
+
+- Visual scanning of the LDFC spectrograms allows for fast checking of false-positive and negatives and has led to the discovery of new species for a given location.  
+
+These advantages come with a serious limitation – the species of interest is only detected where its calls are the dominant acoustic activity in its frequency band over the minute. Surprisingly, the technique has proved to be extremely useful, despite this limitation.
+
+---
+
+## Slide 37: Detecting bats
+Given the low resolution of the LDFC technique as originally developed we had no expectation that would prove useful for identification of echolocating micro-bats. However, we noticed the "red/orange" events in the late evening of several days in March 2016 (see the below stacked ribbon plot) and eventually identified the source as the *white-striped, free-tailed bat*.
+{% include figure.html src="Slides/Slide41.png" caption="Slide 41." %}
+
+---
+
+## Slide 38: Detecting bats
+Here is the same bat call in more detail. This bat is unusual in that the lower range of its call comes below the Nyquist of 11025 hz that we typically use for LDFC spectrograms.  
+
+{% include figure.html src="Slides/Slide42.png" caption="Slide 42." %}
+
+---
+
+## Slide 39: Detecting bats
+{% include figure.html src="Slides/Slide43.png" caption="Slide 43." %}
+
+In another set of night-time recordings using SM4 recording units at their maximum sampling rate of 96 kHz (Nyquist = 48 khz) we could detect all four species of bat present in those recordings. The above LDFC spectrogram shows the traces of the bat calls and what they look like in standard scale spectrograms. We were able to detect 54% of the 310 echolocation passes. This detection rate may seem low, but it is to be remembered that the majority of the passes consisted of low quality calls (see inset, bottom left). 
```

Comment:
"detect all four species" it sounds like there are only four species, remove the word "all".